src/reducers/initialState.js

```diff
@@ -0,0 +1,6 @@
+const signupInitialState = {
```

Comment:
Let's have the initial state as:

```
export default {
  signUp: {
    loading: false,
    success: false,
  }
}
```

So we will be exporting on esingle object that contains all our states. Thus if we need to add more states, we just add more properties to that object, for example:

```
export default {
  signUp: {
    loading: false,
    success: false,
  },
  articles: {},
  user: {},
  ...
}
```